src/components/NftPreview/NftPreview.tsx

```diff
@@ -104,14 +114,19 @@ const StyledNftPreview = styled.div<{ columns: number }>`
     opacity: 1;
   }
 
-  // use margin to create row-gap for now
-  width: 100%;
   margin-bottom: 40px;
+`;
 
-  // width looks nasty but it allows us to conditionally apply different width rules based on # columns:
-  // - if single columm, use hardcoded width because the NFT isnt as wide as the whole page
+const StyledNftPreviewGrid = styled(StyledNftPreview)`
```

Comment:
I extended `StyledNftPreview` to create two different styled components: `StyledNftPreviewGrid` and `StyledNftPreviewList`.

They share the base styles fo the preview nft, the difference is just how the width and margin are set. `StyledNftPreviewGrid` contains the settings we previously used for desktop+tablet+mobile landscape, and `StyledNftPreviewList` contains the settings we previously used for mobile portrait.